src/rpp/rpp/sources/timer.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <rpp/sources/fwd.hpp>
+
+#include <rpp/operators/take.hpp>
+#include <rpp/sources/interval.hpp>
+
+namespace rpp::source
+{
+/**
+ * @brief Creates rpp::observable that emits an integer after a given delay, on the specified scheduler.
+ *
+ * @marble timer
+   {
+       operator "timer(1s)": +--0|
+   }
+ *
+ * @param when time point when the value is emitted 
+ * @param scheduler the scheduler to use for scheduling the items
+ *
+ * @ingroup creational_operators
+ * @see https://reactivex.io/documentation/operators/timer.html
+ */
+template<schedulers::constraint::scheduler TScheduler>
+auto timer(rpp::schedulers::duration when, TScheduler&& scheduler)
```

Comment:
BTW: @CorentinBT, isn't timer difference from interval that it accepts timepoint instead of duration??